llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -754,7 +783,7 @@ static int readModRM(struct InternalInstruction *insn) {
       return base + index;                                                     \
     case TYPE_R8:                                                              \
       index &= mask;                                                           \
-      if (index > 0xf)                                                         \
+      if (index > MAX_GPR_NUM)                                                 \
         *valid = 0;                                                            \
       if (insn->rexPrefix && index >= 4 && index <= 7) {                       \
         return prefix##_SPL + (index - 4);                                     \
```

Comment:
Are they also support in APX?

Reply:
Low-bit registers are also supported while high-bit registers are not.